src/App.tsx

```diff
@@ -1,41 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Just a kind reminder.
React v.17 does not need to be in scope in JSX files. So you have no need to use `React` import.
> [Official docs](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports)

Reply:
You can use all hooks and types as named imports from `react`.